### DISCUSSION TITLE
Folders in Reading Lists and Reading List Sort

### DISCUSSION THREAD

**Discussion by ch13696** [Kareadita/Kavita#2931] [Ideas]
### Idea Description

Comics tend to have a lot of events that better read chronologically. So I think it would be a good idea to add a sort option based on the starting date of the reading lists. It would also be beneficial for us to have folders as well. Marvel's events are usually divided up between Avengers, X-Men, and street level characters (e.g. Luke Cage, Daredevil). It would also help us separate Marvel and DC events as well.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

2 years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by maximo1984** [comment]
I support this. Any and all features that allow users to arrange their content the better; including adding reading lists to collections. This would allow adding TPB reading lists from specific eras to a master collection (New 52, DC Rebirth, Modern DC) for example.
